cmd/frontend/graphqlbackend/search_results.go

```diff
@@ -1317,13 +1317,24 @@ func getPatternInfo(q query.QueryInfo, opts *getPatternInfoOptions) (*search.Tex
 		Languages:                    languages,
 		PathPatternsAreCaseSensitive: q.IsCaseSensitive(),
 		CombyRule:                    strings.Join(combyRule, ""),
+		Index:                        indexValue(q),
 	}
 	if len(excludePatterns) > 0 {
 		patternInfo.ExcludePattern = searchrepos.UnionRegExps(excludePatterns)
 	}
 	return patternInfo, nil
 }
 
+// indexValue parses index:yes (default), index:only, and index:no in the search
+// query.
```

Comment:
Nit

```suggestion
// indexValue converts the query index field to one of yes (default), only, or no enum values.
```

or something, since `indexValue` doesn't do any parsing.